README.md

```diff
@@ -1 +1,98 @@
-# Rye
\ No newline at end of file
+# Rye
+
+[![Carthage Compatible](https://img.shields.io/badge/carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+![Plaforms](https://img.shields.io/badge/platforms-iOS%20-lightgrey.svg)
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/nodes-ios/Rye/blob/master/LICENSE)
+[![CircleCI](https://circleci.com/gh/nodes-ios/Rye.svg?style=shield)](https://circleci.com/gh/nodes-ios/Rye)
+
+## Intro
+
+Rye will enable you to display non intrusive 'Toast' type alerts to your users. 
```

Comment:
Should it be " instead of ' 

"Toast" instead of 'Toast'

Also, consider

Rye allows you to present non intrusive "Toast" type alerts to your users